.golangci.yaml

```diff
@@ -21,6 +15,11 @@ linters:
   - errcheck
 
 linters-settings:
+  # This stanza is needed to avoid getting the warning
+  # WARN [config_reader] The configuration option `linters.errcheck.ignore` is deprecated, please use `linters.errcheck.exclude-functions`.
+  errcheck:
+    ignore: ''
```

Comment:
Let me find the reference for this. Essentially, there's a bug in golangci atm where for some reason that deprecated field is set to something by default. So, we need to set it to empty to avoid the warning.

Reply:
Would be good to have a link to a bug in the comment.

If it is a bug in golangci and if it is just a warning which does not impact the result of the job - then I would just leave it as is. I suspect it will get resolved once we upgrade to a version which resolves the bug.

Reply:
I'll add it it the next one to skip CI